borg/archiver.py

```diff
@@ -1076,6 +1107,34 @@ def build_parser(self, args=None, prog=None):
         subparser.add_argument('location', metavar='REPOSITORY', nargs='?', default='',
                                type=location_validator(archive=False))
 
+        subparser = subparsers.add_parser('key-export', parents=[common_parser],
```

Comment:
It seems moving argparsing into keymanager is too intrusive for 1.0-maint for Thomas.

We still should decide if having a "key" command and subcommands for import and exports is worth the style change from the existing debug commands.